recipe_scrapers/_utils.py

```diff
@@ -33,6 +33,20 @@
 
 SERVE_REGEX_TO = re.compile(r"\d+(\s+to\s+|-)\d+", flags=re.I | re.X)
 
+RECIPE_YIELD_TYPES = (
+    ("dozen", "dozens"),
+    ("batch", "batches"),
+    ("cake", "cakes"),
+    ("sandwich", "sandwiches"),
+    ("bun", "buns"),
+    ("cookie", "cookies"),
+    ("muffin", "muffins"),
+    ("cupcake", "cupcakes"),
+    ("loaf", "loaves"),
+    ("pie", "pies"),
```

Comment:
This is a nice feature :+1: 

Slightly off-topic: I have some ideas about how to try to find and populate more values here -- and perhaps similar techniques could be used to improve the quality of `get_minutes` matching.  Both of them are brief snippets of text containing numbers interspersed with words (although _more than one_ number is common in the time case).

Roughly speaking: if we could collect a two-column spreadsheet / database table containing unparsed texts and their parsed equivalents, I think that would help us to identify additional parse-words (more serving units to add here, for example) and also parsing errors.